includes/functions.php

```diff
@@ -4274,3 +4271,48 @@ function dokan_get_new_product_url() {
         dokan_get_navigation_url( 'new-product' )
     );
 }
+
+/**
+ * Dokan new seller enable selling statuses.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @return array
+ */
+if ( ! function_exists( 'dokan_new_seller_enable_selling_statuses' ) ) {
+    function dokan_new_seller_enable_selling_statuses() {
+        return apply_filters(
+            'dokan_new_seller_enable_selling_statuses', [
+                'automatically' => __( 'Automatically', 'dokan-lite' ),
+                'manually'      => __( 'Manually', 'dokan-lite' ),
+            ]
+        );
+    }
+}
+
+/**
+ * Get new seller selling status setting.
+ * We are placing this function here because this function may access from admin and front-end both.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @param string $status
+ *
+ * @return string
+ */
+if ( ! function_exists( 'dokan_get_new_seller_enable_selling_status' ) ) {
```

Comment:
Extract these two functions into a utility class like `AdminSettings` and also use from container